build_tools/packaging/linux/fetch_artifacts.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+"""Given ROCm artifacts directories, performs the download and extraction of artifacts
+
+```
+./fetch_artifacts.py --artifact-dir ./output-linux-portable/build/artifacts \
+        --gfx-arch gfx94X 
+```
+"""
+
+import urllib.request
+import tarfile
+import shutil
+#from pathlib import Path
+#import sys
+import os
+
+from packaging_utils import *
+
+# Directory for downloading tar artifacts   
+ARTIFACTS_DOWNLOAD_DIR = f"{os.getcwd()}/artifacts_tar"
```

Comment:
If keeping the new script, prefer pathlib over `os`. There is a list of corresponding APIs at https://docs.python.org/3/library/pathlib.html#corresponding-tools

Reply:
Done